media/js/firefox/firstrun/firstrun_quantum.js

```diff
@@ -18,6 +18,7 @@
         scene.dataset.animate = 'true';
         var hideOrShowSkipButton = function (data) {
             switch(data.data.url) {
+            case '':
```

Comment:
FxA's "enter email" page is the index (/) and has a url of `''` (a blank string).